src/Illuminate/Support/helpers.php

```diff
@@ -653,6 +653,21 @@ function str_plural($value, $count = 2)
 	}
 }
 
+if ( ! function_exists('pluralize'))
+{
+	/**
+	 * Attempts to pluralize the singular word unless count is 1.
```

Comment:
I can copy all the descriptions given by the Rails docs or you think its better to modify this line?

> Attempts to pluralize the singular word unless count is 1. If plural is supplied, it will use that when count is > 1, otherwise it will use the Inflector to determine the plural form.